api/src/test/java/org/apache/iceberg/TestIcebergBuild.java

```diff
@@ -37,6 +41,34 @@ public void testFullVersion() {
                 + ")");
   }
 
+  @Test
+  public void testVersionNotUnspecified() {
+    assertThat(IcebergBuild.version()).isNotEqualTo("unspecified");
+  }
+
+  @Test
```

Comment:
@ajantha-bhat Added this test, all good?

Reply:
Thanks